docs/consume-packages/PackageDownload-Functionality.md

```diff
@@ -0,0 +1,76 @@
+
+`PackageReference` allows you to manage your package dependencies directly in your project file.
+When you run restore, the transitive dependencies are resolved automatically and the applicable references are chosen for each package in the project graph.
+
+In [5.3 of NuGet](..\release-notes\NuGet-5.3.md) a companion functionality was introduced for [.NET SDK-style projects](..\resources\check-project-format.md) called `PackageDownload`.
+
+## PackageDownload specification
+
+PackageDownload is a utility feature for all .NET SDK-style projects, and it works along side `PackageReference`.
+
+`PackageDownload` items support different attributes compared to `PackageReference`. Only attributes listed in the below table are supported.
+
+| Attributes | Description | Example |
+|------------|-------------|---------|
+| Version | Only exact versions, surrounded with `[]` are supported. Multiple versions can be specified separated by `;` | `[1.0.0]`, `[1.0.0];[2.0.0]` |
+
+Packages acquired through PackageDownload will undergo the same [installation process](..\concepts\package-installation-process.md) as packages acquired through PackageReference.
+This means [package signatures](installing-signed-packages.md) are validated, [package source mapping](Package-Source-Mapping.md) is considered.
+All newly acquired PackageDownload packages will be installed in the global packages folder.
+
+| Feature | PackageReference | PackageDownload |
+|-|------------------|-----------------|
+| Package assets selection | Assemblies from packages are automatically added to the project and can be used for compile and runtime | Assemblies or any other assets from the package are not included in the project. |
+| Dependencies | Automatically resolved, and flattened to a single version | Not considered at all |
+| pack | Included in the package specification | Not included in the package specification. |
+| Transitivity | PackageReference items are automatically propagated to dependant projects | PackageDownload items are ignored by dependant projects |
+| Version | Version ranges such as `1.0.0` or `[1.0.0, )` are supported. Exactly 1 version is allowed. | Only exact versions are supported. More than 1 version can be downloaded. |
+| dotnet list package | All dependencies are included | PackageDownload packages are not shown by `dotnet list package`. |
+
+Due to the fact that PackageDownload are not tied to the project in any way beyond acquisition, multiple versions of the same package can be downloaded.
+
+### PackageDownload limitations
+
+Given that this is an advanced feature with limited applicability, it doesn't have a tooling support equivalent to PackageReference.
+
+- There is no VisualStudio or dotnet.exe functionality to modify PackageDownload items. You can only change them manually in your project files.
+- dotnet add, remove, and list commands do not account for PackageDownload items.

```

Comment:
Do lock file take PackageDownload into consideration?  If not, this would be an important limitation to list considering some customers use lock files & content hash validation as a layer of supply chain security.

Reply:
No, they do not. 

That's a great point and something I should have added.